app/models/container_template.rb

```diff
@@ -26,15 +27,16 @@ class ContainerTemplate < ApplicationRecord
     "Service"               => ContainerService,
   }.freeze
 
-  def instantiate(params, project = nil)
+  def instantiate(params, labels, project = nil)
```

Comment:
we can use template.object_labels as default value, if you prefer? (I thought that we always want to add a unique template label to mark the objects)